cli/src/cmd/test.js

```diff
@@ -2,6 +2,7 @@ import { Parser } from 'yargs/helpers';
 import { rm, mkdir } from 'node:fs/promises';
 import { join } from 'node:path';
 import { hexlify, randomBytes } from 'ethers';
+import * as fs from 'fs/promises';
```

Comment:
duplicated import of `fs/promises`, prefer importing methods you need

```suggestion
import { rm, mkdir, readdir, stat} from 'node:fs/promises';
import { join } from 'node:path';
import { hexlify, randomBytes } from 'ethers';
```